mrt/mail.py

```diff
@@ -123,6 +125,9 @@ def send_notification_message(recipients, participant):
 @registration_signal.connect_via(ANY)
 def send_registration_message(sender, participant):
     sender = app.config['DEFAULT_MAIL_SENDER']
```

Comment:
Shouldn't this also be a ```get_default_sender()``` call? or the app admin is supposed to receive these emails instead of the meeting owner?